src/test/scala/com/dimafeng/testcontainers/integration/GenericContainerSpec.scala

```diff
@@ -20,3 +21,11 @@ class GenericContainerSpec extends FlatSpec with ForAllTestContainer {
     ).mkString.contains("If you see this page, the nginx web server is successfully installed"))
   }
 }
+
+class GenericContainerDockerFileSpec extends GenericContainerSpec {
+  private val imageFromDockerfile = new ImageFromDockerfile().withFileFromClasspath("Dockerfile", "generic-container-dockerfile")
```

Comment:
In this line, the second string is the name of the Dockerfile, but what is the first string -- `"Dockerfile"`?

Reply:
The first string is the file location on the 'virtual filesystem' that is used when building the Docker container, I think. see: https://github.com/testcontainers/testcontainers-java/blob/master/docs/usage/dockerfile.md#dockerfile-from-string-file-or-classpath-resource